test/distributed_exec.jl

```diff
@@ -3,6 +3,10 @@
 using Test, Distributed, Random, Serialization, Sockets
 import Distributed: launch, manage
 
+if haskey(ENV, "CI")
+    @test !occursin("stdlib", pathof(Distributed))
```

Comment:
Hm will this work if it is run as part of Base's test-suite?

Reply:
How do I detect if I'm running within the Base test-suite?

Reply:
Nvm, I've just added a variable to GHA CI that we can detect